app/controllers/repositories_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
   def show
     authorize @repository
     @tags = @repository.tags.order("created_at DESC")
+    @repository_comments = @repository.comments.page(params[:comments_page]).per(5)
```

Comment:
I'm actually not sure about paginating comments. Neither Github nor Gitlab, nor Docker Hub does it. I don't expect a *ton* of comments to be created honestly. I think that this can be simplified. If you agree, then get rid off the pagination part on the view of this method.